test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/DDRValueTypeTest.java

```diff
@@ -61,10 +61,8 @@ private static void createAndCheckValueType() throws Throwable {
 		Object assortedValueWithSingleAlignment = ValueTypeTests.createAssorted(makeAssortedValueWithSingleAlignment, ValueTypeTests.typeWithSingleAlignmentFields);
 		Object assortedValueWithSingleAlignmentAlt = ValueTypeTests.createAssorted(makeAssortedValueWithSingleAlignment, ValueTypeTests.typeWithSingleAlignmentFields, altFields);
 		Object valueTypeWithVolatileFields = ValueTypeTests.createValueTypeWithVolatileFields();
-		
-		Object[] valArray = ValueClass.newNullRestrictedArray(assortedValueWithSingleAlignmentClass, 2);
-		valArray[0] = assortedValueWithSingleAlignment;
-		valArray[1] = assortedValueWithSingleAlignmentAlt;
```

Comment:
Please keep this line since it is not set to the default value.

Reply:
Restored